src/cache.rs

```diff
@@ -431,7 +448,7 @@ impl Cache {
     fn find_existing(&self, resource: &str, subdir: Option<&str>) -> Vec<Meta> {
         let mut existing_meta: Vec<Meta> = vec![];
         let glob_string = format!(
-            "{}.*.meta",
+            "{}*.meta",
```

Comment:
I don't know if this was intentional or not, but the glob pattern before wasn't matching against the `.meta` files. Changing to this fixed it, and the files stopped being downloaded again.

Reply:
Huh, that looks like a bug. Good catch!